### PR TITLE
Flogo CLI plugin for Flogo Enterprise builder

### PR DESCRIPTION
**Pull request by lixingwang**

### CODE REVIEW

File: cli/plugins/builder/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+Preparation:
+
+     1. Install Flogo CLI first, please following CLI guideline
+     2. Set FLOGO_HOME Env Variable, such as: 
+        export FLOGO_HOME=/opt/tibco/flogo/2.6
+
+Installation
+    
+      1. Install plugin: flogo plugin install github.com/TIBCOSoftware/tci-flogo/cli/plugins/builder
```
Comment (vijaynalawade) [inline cli/plugins/builder/README.md]: I think ```github.com/TIBCOSoftware/tci-flogo/project-flogo/cli/plugin``` would be better. @rishikeshpalve ??

File: cli/plugins/builder/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+[Flogo CLI](https://github.com/project-flogo/cli) Plugin for Flogo Enterprise Builder
```
Comment (vijaynalawade) [inline cli/plugins/builder/README.md]: ```Project Flogo CLI Plugin for Flogo Enterprise```

File: cli/plugins/builder/README.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    
+      1. Install plugin: flogo plugin install github.com/TIBCOSoftware/tci-flogo/cli/plugins/builder
+      2. Just update plugin if you want to update flogo plugin update github.com/TIBCOSoftware/tci-flogo/cli/plugins/builder
+      
+Usage:
+    
+    The plugin wrapper Flogo Enterprise builder, so all build options will available at this command as well.
+   
+     Example:
+     flogo febuilder build --f ~/Downloads/log.json
```
Comment (vijaynalawade) [inline cli/plugins/builder/README.md]: ```flogo enterprise build --f </app/json>```
@rishikeshpalve ??

File: cli/plugins/builder/cmd.go [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+	fePluginCommand.Flags().StringVar(&file, "file", "", "Path to Flogo application json file")
+	fePluginCommand.Flags().StringVar(&file, "f", "", "Path to Flogo application json file")
+
+	fePluginCommand.Flags().StringVar(&name, "name", "", "Name of the binary. If not provided, binary with name <APPNAME>-<OS_NAME>-<OS_ARCH> will be generated")
+	fePluginCommand.Flags().StringVar(&name, "n", "", "Name of the binary. If not provided, binary with name <APPNAME>-<OS_NAME>-<OS_ARCH> will be generated")
+
+	fePluginCommand.Flags().StringVar(&output, "output", "", "Folder where binary to be created. By default, binary will be created in current directory")
+	fePluginCommand.Flags().StringVar(&output, "o", "", "Folder where binary to be created. By default, binary will be created in current directory")
+
+	fePluginCommand.Flags().StringVar(&platform, "platform", "", "Build binary for specific platform. Value must be specified in the form of <OS_NAME>/<OS_ARCHITECTURE> e.g. linux/386, linux/amd64, By default, current OS is used")
```
Comment (vijaynalawade) [inline cli/plugins/builder/cmd.go]: We should also support ```GOOS``` and ```GOARCH``` environment vars.

Comment (lixingwang) [reply cli/plugins/builder/cmd.go]: It supports by native,  Go Build will take GOOS and GOARCH env var while building the binary

Comment (lixingwang) [reply cli/plugins/builder/cmd.go]: This has been fixed at FE builder side

File: cli/plugins/builder/README.md (hunk 1 above)
Comment (lixingwang) [reply cli/plugins/builder/README.md]: Changed to  github.com/TIBCOSoftware/tci-flogo/project-flogo/cli/plugins/builder

File: cli/plugins/builder/README.md (hunk 3 above)
Comment (lixingwang) [reply cli/plugins/builder/README.md]: now it will be: flogo enterprise build -f

**Pull request merged**